### ISSUE TITLE
WebDAV support / Web folder

### ISSUE THREAD

**Issue by bleroy** [bleroy/OrchardImportTest#4]
Hi, You should support WebDAV for online editting of office documents like
sharepoint or Web folders.  

Work Item Details
--------------------
**Original CodePlex Issue:**	[Issue 14757](http://orchard.codeplex.com/workitem/14757)
**Status:**	Closed
**Reason Closed:**	Unassigned
**Assigned to:**	Unassigned
**Reported on:**	Nov 11, 2009 at 11:55 AM
**Reported by:**	dadvav
**Updated on:**	Nov 11, 2009 at 6:47 PM
**Updated by:**	BertrandLeRoy
**Closed on:**	Nov 11, 2009 at 6:47 PM

**Comment by bleroy** [comment]
On *2009-11-12 02:45:33 UTC*, **BertrandLeRoy** commented:

Hi, 

Thanks for the suggestion. Document management is not a goal of the project at this time. Once the extensibility model is in place, it will be possible to develop something like this as an extension package.


Cheers, 
Bertrand
